src/toast/schedule_sim_ground.py

```diff
@@ -2893,6 +2904,11 @@ def parse_args(opts=None):
         required=False,
         help="Fixed observing elevations in a comma-separated list.",
     )
+    parser.add_argument(
```

Comment:
Adding the offset as a command line argument is not ideal. We want to add different offsets to different targets so that the scheduler can treat calibration scans of each wafer separately. The offset should be recorded as additional fields to the SSO patch.